apps/emqx_conf/src/emqx_cluster_rpc.erl

```diff
@@ -388,7 +413,8 @@ read_next_mfa(Node) ->
                 }),
                 TnxId;
             [#cluster_rpc_commit{tnx_id = LastAppliedID}] ->
-                LastAppliedID + 1
+                OldestId = get_oldest_mfa_id(),
+                max(LastAppliedID + 1, OldestId)
```

Comment:
If the commit's tnxid is clean up in cluster_mfa, we should follow up the oldest mfa_id.